packages/commonwealth/client/scripts/views/pages/contracts/add_contract_form.tsx

```diff
@@ -1,168 +0,0 @@
-/* @jsx m */
```

Comment:
Why was this page deleted?

Reply:
this should not have been included in the first contracts main PR. I forgot to remove this! It is used nowhere